js/terms.json

```diff
@@ -136,52 +131,320 @@
     "term": "Special election",
     "definition": "A primary, general or runoff election that is not a regularly scheduled election and that is held to fill a vacant seat in the House of Representatives or the Senate. 100.2(f)."
   },
-   {
+  {
     "term": "Separate segregated fund (SSF)",
     "definition": "A political committee established, administered or financially supported by a corporation or labor organization, popularly called a Corporate or Labor Political Action Committee (PAC). See 114.1(a)(2)(iii). The term \"financially supported\" does not include contributions to the SSF, but does include the payment of establishment, administration or solicitation costs. 100.6(b)."
```

Comment:
Remove `See...` before citations? Most of the following changes don't include it before the list starts